components/director/internal/domain/application/resolver.go

```diff
@@ -265,17 +266,16 @@ func (r *Resolver) Applications(ctx context.Context, filter []*graphql.LabelFilt
 	}, nil
 }
 
-// Application missing godoc
-func (r *Resolver) Application(ctx context.Context, id string) (*graphql.Application, error) {
+func (r *Resolver) getApplication(ctx context.Context, get func(context.Context) (*model.Application, error)) (*graphql.Application, error) {
```

Comment:
Can you put this func somewhere at the bottom right after the exported functions starting with a capital letter?